Documentation/dont_submit

```diff
@@ -0,0 +1 @@
+conversion_tools
```

Comment:
It is probably better that the line ends with an end-of-line, otherwise the file can be considered as a binary file.